api/orchestrator/orchestrator.proto

```diff
@@ -824,3 +824,14 @@ message State {
   // Reference to the certificate
   string certificate_id = 5;
 }
+
+// ToeStatus represents the status of the Target of Evaluation. 
+// With that status the ToE hook function can be informed about the reason of its call. 
+message ToeStatus {
```

Comment:
I think the other message are named similar to `ChangeEvent`, status might be a bit misleading.
Also `0` must be `_UNSPECIFIED` according to the style guide and the rest of the enums should have similar names like the enum itself. See https://developers.google.com/protocol-buffers/docs/proto3#enum

We probably don't follow this everywhere (yet).